cfme/configure/access_control/__init__.py

```diff
@@ -878,14 +878,15 @@ def create(self, description=None, role=None, tenant="My Company", ldap_credenti
         else:
             view.add_button.click()
             flash_message = 'Group "{}" was saved'.format(group.description)
-        view = self.create_view(AllGroupView, wait='10s')
 
         try:
             view.flash.assert_message(flash_blocked_msg)
             raise RBACOperationBlocked(flash_blocked_msg)
         except AssertionError:
             pass
 
+        view = self.create_view(AllGroupView, wait='10s')
+
```

Comment:
Looks good , we can remove blank line below view though.

Reply:
Removed.